include/nbl/video/CJITIncludeLoader.h

```diff
@@ -8,10 +8,12 @@
 
 #include <string>
 
+#include "nbl/builtin/hlsl/type_traits.hlsl"
+
+#include "glm/gtx/string_cast.hpp"
```

Comment:
not sure I want this floating around in public headers, can you make the specialization that uses `glm::to_string` not be inline and have it defined in the `.cpp` file instead?

Reply:
please no more 3rdparty headers in public headers

Reply:
todo